src/Lean/Parser/Command.lean

```diff
@@ -754,6 +754,30 @@ list, so it should be brief.
   optional (docComment >> ppLine) >>
   "tactic_extension " >> ident
 
+/--
+Document a recommended spelling for a notation in identifiers.
+
```

Comment:
A little bit about the purpose/context for this at the top of the hover might help users who are encountering it for the first time figure out whether they should spend time understanding it. Perhaps something along the lines of this?
 
```suggestion
Theorems should generally be systematically named after their statement, rather than creatively. Non-identifier notations should be referred to consistently by their recommended spelling.

```

Reply:
I like it, thanks.